docs/dev/testing.rst

```diff
@@ -261,3 +262,45 @@ database containing all years of FERC 1 data (equivalent to ``tox -e ferc1_schem
 .. code-block:: console
 
   $ pytest --etl-settings test/settings/full-integration-test.yml test/integration/etl_test.py::test_ferc1_schema
+
+-------------------------------------------------------------------------------
+Data Validation
+-------------------------------------------------------------------------------
+Once the PUDL ETL pipeline has processed all of the available data, we have a
+collection of tests that can be run to verify a collection of expectations
+about what the outputs should look like. These data validation tests are
+organized into datasource specific modules under ``test/validate``. Running
+the full data validation can take as much as an hour, depending on your
+computer.
+
+These tests require a fully populated PUDL database which contains all
+available FERC and EIA data, as specified by the
+``test/settings/full-integration-test.yml`` input file. They are run against the
+"live" SQLite database in your pudl workspace at ``sqlite/pudl.sqlite``. To run
+the full data validation against an existing database:
+
+.. code-block:: console
+
+    $ tox -e validate
+
+Which internally calls pytest like this, but in a clean Python environment:
+
+.. code-block:: console
+
+    $ pytest --live-dbs test/validate
+
+The data validation cases that pertain to the contents of the data tables are
+currently stored as part of the :mod:`pudl.validate` module.
+
+The expected number of records in each output table is stored in the validation
+test modules under ``test/validate`` as pytest parameterizations.
+
+Data Validation Notebooks
+^^^^^^^^^^^^^^^^^^^^^^^^^
+We also have a collection of Jupyter Notebooks that use the same functions as
+the data validation tests and also produce some visualizations of the data to
+make it easier to understand what's wrong when validation fails. These
+notebooks are stored in ``test/notebooks``
```

Comment:
Clarify the wording of this sentence. Gets a little jumbled during the `and also produce` portion. Maybe change it to something like `...use the same functions as the data validations tests to produce visualizations of the data.....` It might even serve to lead the sentence with the use case. For ex: `To better understand failed validation tests...we have notebooks!`

Reply:
Okay, I split it into two sentences.